config/base.cfg

```diff
@@ -8,6 +7,7 @@ extends =
 show-picked-versions = true
 newest = false
 parts =
+    instance
```

Comment:
```suggestion
```
This should be pulled in by some other stuff

Reply:
ah, that's interesting. didn't know that.